test/rulesets/Random/random.jl

```diff
@@ -19,4 +19,27 @@
             @test all(map(x -> x isa Zero, pb(10)))
         end
     end
+
+    @testset "rand" begin
+        non_differentiables = [((), Float64),
+                               ((Random.default_rng(),), Float64),
+                               ((Random.default_rng(),2,2), Matrix{<:Float64}),
+                               ((Float32,), Float32),
+                               ((Float32,2,2), Matrix{<:Float32}),
+                               ((Float32,(2,2)), Matrix{<:Float32}),
+                               ((2,2), Matrix{<:Float64})]
```

Comment:
Just a style thing -- not sure that BlueStyle addresses this particular construction explicitly, but below is "morally" the kind of thing that the style guide expects

```suggestion
        non_differentiables = [
            ((), Float64),
            ((Random.default_rng(),), Float64),
            ((Random.default_rng(),2,2), Matrix{<:Float64}),
            ((Float32,), Float32),
            ((Float32,2,2), Matrix{<:Float32}),
            ((Float32,(2,2)), Matrix{<:Float32}),
            ((2,2), Matrix{<:Float64}),
        ]
```